internal/super/generate.go

```diff
@@ -188,15 +207,47 @@ fun main() {
 		return nil, fmt.Errorf("error creating directories: %w", err)
 	}
 
+	// Write files
 	err = state.ReaderWriter().WriteFile(filenameWithBasePath, []byte(fileToWrite), 0644)
 	if err != nil {
 		return nil, fmt.Errorf("error writing file: %w", err)
 	}
 
 	logger.Info(fmt.Sprintf("Generated new %s: %s at %s", templateType, name, filenameWithBasePath))
 
+	if generateFlags.SkipTests != true {
+		if _, err := state.ReaderWriter().ReadFile(testFilenameWithBasePath); err == nil {
```

Comment:
should we instantiate thes `testFilenameWithBasePath` and `testBasePath` bits inside here instead? Maybe as  seperate function that this calls so we don't bloat this conditional?